catalog/app/components/JsonEditor/State.js

```diff
@@ -120,15 +117,20 @@ function convertType(value, typeOf) {
   }
 }
 
+const EmptySchema = {}
```

Comment:
let's use lower camelCase or uppercase for this, so it looks like a regular var or a constant, and reserve upper CamelCase for classes, components and types